src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -136,6 +137,13 @@ const TransformerMap: {
     },
   },
   [IDX_STEP.CHALLENGE_AUTHENTICATOR]: {
+    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: {
+      transform: transformSymantecVipAuthenticator,
+      buttonConfig: {
+        showDefaultSubmit: false,
+        showVerifyWithOtherLink: true,
+      },
+    },
```

Comment:
nit: place this lower for alphabetical order, same with the enroll authenticator version

Reply:
done